src/elements/content-sidebar/__tests__/SidebarNav-test.js

```diff
@@ -81,18 +81,10 @@ describe('elements/content-sidebar/SidebarNav', () => {
         ).toContain('bcs-nav-btn-is-selected');
     });
 
-    test('should render additional tabs', () => {
+    test('should render the additional tabs loading state', () => {
         const props = {
             hasAdditionalTabs: true,
-            additionalTabs: [
-                {
-                    id: 200,
-                    title: 'Test title',
-                    iconUrl: 'https://foo.com/icon',
-                    callback: jest.fn(),
-                    status: 'ADDED',
-                },
-            ],
+            additionalTabs: [],
         };
         const wrapper = getWrapper(props);
         expect(wrapper.find(AdditionalTabs)).toHaveLength(1);
```

Comment:
Does this mean that the wrapper should exist, but have no children and be in a loading state? It might be better to test those things explicitly rather than using a mounted snapshot.

Reply:
I'm going to remove this condition. I think here I'll just test that the loading state renders, and inside the components i'll test the loaded state.